retrieval/transform.go

```diff
@@ -116,7 +116,7 @@ func (b *sampleBuilder) next(ctx context.Context, samples []tsdb.RefSample) (*mo
 		point.Value = &monitoring_pb.TypedValue{
 			Value: &monitoring_pb.TypedValue_DistributionValue{v},
 		}
-		return &ts, 0, samples, nil
+		return &ts, entry.hash, samples, nil
```

Comment:
Great catch. Can we have a unit test for this?

Reply:
unit test added.